index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Template1</title>
+</head>
+
+<body>
+    <main>
+
+        <header>
+            <nav class="navbar">
+                <p class="logo">MyLogo</p>
+                <div class="hamburger">
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                    <div class="bar"></div>
+                </div>
+            </nav>
+        </header>
+        <section class="headline">
+
+            <section class="primtext">
+                I’m Mariane Glad to see you!
+            </section>
+
+            <section class="content">
+                I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </section>
+
+            <footer class="connect">
+                <p>LET’S CONNECT</p>
+                <img src="social-media-icons.png" alt="social media icons">
```

Comment:
Please Implement the button interactions (enable, hover, etc.) for your social media icons.